h2o-algos/src/test/java/hex/tree/gbm/GBMTest.java

```diff
@@ -115,18 +123,19 @@ Constraints emptyConstraints(Frame f) {
     }
   }
 
-  @Test public void testOneHotExplicitWithPOJO() {
+  @Test public void testOneHotExplicitWithPOJO() throws Exception {
     try {
       Scope.enter();
       final String response = "CAPSULE";
-      Frame fr = parse_test_file("./smalldata/logreg/prostate.csv")
+      final String testFile = "./smalldata/logreg/prostate.csv";
+      Frame fr = parse_test_file(testFile)
```

Comment:
Minor: can also be final. Together with some other variables. Not something I insist on, yet I noticed it and want to mention it.